lib/classes/Swift/Transport/AbstractSmtpTransport.php

```diff
@@ -231,6 +231,31 @@ public function stop()
     }
 
     /**
+     * Check if this Transport mechanism is still alive.
```

Comment:
You need to either copy over the phpdoc, or use the correct inherit doc stuff otherwise the description from the other doc isn't applied here